chunks.go

```diff
@@ -91,6 +93,8 @@ func newChunkWriter(dir string) (*chunkWriter, error) {
 	return cw, nil
 }
 
+// XXX: should this be called head instead of tail? We also call the
+// currently active block the "head" instead of the "tail".
```

Comment:
I think head makes most sense to me intuitively. When I'm thinking about head+tail, I'm thinking of the classic snake game, where the head is at the front and basically moving forwards, and old stuff at the tail gets cut off :)